sdcclient/_scanning.py

```diff
@@ -323,6 +307,49 @@ def get_pdf_report(self, image, tag=None, date=None):
 
         return [True, res.content]
 
+    def import_image(self, infile):
```

Comment:
Is there any example (maybe not just using this function) to show how this functionality can be used? I think it can expose very interesting use cases.

Reply:
@davideschiera this function is part of the inline-scan solution which is currently in a bash script. This function essentially implements the import image API being used in the script here https://github.com/sysdiglabs/secure-inline-scan/blob/inline-scan/inline_scan.sh#L463

Reply:
Can an example (similar to examples/get_anchore_users_account.py) show how to import an image and then do something with it (eg. scan it)?

I was imagine something like that, but I'm not sure if it can be meaningful or useful.